certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -3985,418 +4992,1389 @@
+                            "description": "Product Type."
+                        },
+                        "purchaseDate": {
+                            "type": "string",
+                            "description": "Purchase Date."
+                        },
+                        "additionalInfo": {
+                            "type": "string",
+                            "description": "Additional Info."
+                        },
+                        "quantity": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Quantity."
+                        },
+                        "itemizedCustomFields": {
+                            "type": "array",
+                            "items": {},
+                            "description": "Itemized Custom Fields."
+                        },
+                        "productCategories": {
+                            "type": "array",
+                            "items": {},
+                            "description": "Product Categories."
+                        }
+                    }
                 },
-                "description": {
-                  "type": "string",
-                  "description": "Description.",
-                  "title": "Description",
-                  "x-ms-visibility": "internal",
-                  "default": "Added by Impexium Connector"
-                }
-              },
-              "required": [
-                "callBackUrl",
-                "description",
-                "secret",
-                "typeName",
-                "filters"
-              ]
+                "description": "Bought Together With."
+            },
+            "registrantTypeCode": {
+                "type": "string",
+                "description": "Registrant Type Code.",
+                "title": "Registrant Type Code"
+            },
+            "registrantTypeName": {
+                "type": "string",
+                "description": "Registrant Type Name.",
+                "title": "Registrant Type Name"
+            },
+            "registrationNumber": {
+                "type": "string",
+                "description": "Registration Number.",
+                "title": "Registration Number"
             }
-          }
-        ]
-      }
-    },
-    "/api/v1/Webhooks/CommitteeMemberUpdated": {
-      "x-ms-notification-content": {
-        "description": "Committee Member Updated.",
-        "schema": {
-          "$ref": "#/definitions/CommitteeMemberPayload"
         }
-      },
-      "post": {
-        "tags": ["Webhook", "Committee"],
-        "responses": {
-          "200": {
-            "description": "Success",
-            "schema": {}
-          }
+    },
+    "CommitteeMemberData": {
+        "type": "object",
+        "properties": {
+            "committee": {
+                "$ref": "#/definitions/CommitteeData"
```

Comment:
Please replace this with the content, adding the `Committe -` prefix to the descriptions